repos/spack_repo/builtin/packages/ambertools/package.py

```diff
@@ -21,7 +21,7 @@ class Ambertools(CMakePackage):
     maintainers("d-beltran")
 
     version("22jlmrcc", sha256="1571d4e0f7d45b2a71dce5999fa875aea8c90ee219eb218d7916bf30ea229121")
-
+    version("v25", sha256="ac009b2adeb25ccd2191db28905b867df49240e038dc590f423edf0d84f8a13b", url="https://ambermd.org/downloads/ambertools25.tar.bz2")
```

Comment:
Just add the `sha256` of the downloaded tarball. How to get that tarball is a different story. The `url` of the package just has the function of providing a means to determine the filename. But I believe this package should also be marked as `manual_download = True`, like in `amber`.

Reply:
So, something like:
```
url = "file://{0}/ambertools25.tar.bz2".format(os.getcwd())
manual_download = True

version("25", sha256="ac009b2adeb25ccd2191db28905b867df49240e038dc590f423edf0d84f8a13b")
```